app/containers/UIKit/MultiSelect/index.tsx

```diff
@@ -26,7 +26,7 @@ import Input from './Input';
 import styles from './styles';
 
 export interface IItemData {
-	value: string;
+	value: any;
```

Comment:
should we type this `value` as `value: { name?: string }; ` ?

Reply:
This is a generic component, that depends on how we manipulate this. Because there are other places that value is only a string, and in this case is this interface that you mentioned!